src/stdune/path.ml

```diff
@@ -156,6 +157,76 @@ end = struct
 
 end
 
+module Relative : sig
```

Comment:
I think we should have an intermediate commit (that builds) that does not have this `Relative` module at all, so there is no unexpected change of semantics.